### ISSUE TITLE
Trojan trigger（/NAD/trigger/best_square_trigger_cifar10.npz） not effective

### ISSUE THREAD

**Issue by coldpark** [bboylyg/NAD#15]
I have been using your open source project NAD for defending against Trojan attacks in deep learning models, and I have noticed a potential issue with the trojan trigger provided in the project.

Specifically, I have found that the trojan trigger provided in the project may not be effective against Trojan attacks, as it can be easily detected and eliminated by a simple fine-tuning process. By generating a teacher model and fine-tuning it on the target dataset, I was able to significantly reduce the ASR of the model with the trojan trigger.

Therefore, I suspect that the trojan trigger provided in the project may not be a true Trojan trigger, or at least may not be effective against sophisticated Trojan attacks. I would appreciate it if you could provide more information or guidance on how to improve the effectiveness of the trojan trigger.

Thank you for your time and attention.

**Comment by bboylyg** [comment]
Hi, this is an interesting question! 

For your questions, I encourage to do experiments as follows:
- Reproducing the Trojan attacks by yourself or using another released;
- Training the Trojan backdoored model with standard data augmentation (flip, crop, and data normalization) and then, finetuning the backdoored model to verify its effectiveness.

**Comment by coldpark** [comment]
I see, thanks for your reply.
